aspnetcore/razor-components/index.md

```diff
@@ -77,51 +46,63 @@ A *component* is a .NET class. The class can either be written directly, as a C#
 
 When this component is used elsewhere in the app, IntelliSense speeds development with syntax and parameter completion.
 
-Components can be:
-
-* Nested.
-* Created with Razor (*\*.cshtml*) or C# (*\*.cs*) code.
-* Shared via class libraries.
-* Unit tested without requiring a browser DOM.
-
-## Infrastructure
+Components render into an in-memory representation of the render tree that can then be used to update the UI in a flexible and efficient way.
 
-Razor Components offers the core facilities that most apps require, including:
+Core facilities that most apps require are provided by the Razor Components framework, including:
```

Comment:
Razor Components support core facilities required by most apps, including:

Reply:
Should these all be links?

Reply:
IMO, not really. These are just **_teasers!_** lol ... and, of course, examples of what's supported. My recommendation is not to have readers clicking off into the middle of detailed technical docs at this point (*very early*) in their engagement with the tech. I'd cross-link them within technical topics ... but not here. My recommendation is to let the reader go down the *Components* topic (and the others in the TOC, e.g., *Layouts*) in due course.